github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml

```diff
@@ -317,6 +317,9 @@ plugins:
   - lgtm
   - owners-label
 
+  RHsyseng/rhcos-slb:
+  - trigger
+
```

Comment:
Add "- override" this is usefull is something is bad at CI but we need something to get merged.